airbyte-integrations/csv-destination/build.gradle

```diff
@@ -3,10 +3,31 @@ plugins {
     id 'com.bmuschko.docker-remote-api'
     id 'application'
 }
+
+sourceSets {
+    integrationTest {
+        java {
+            srcDir 'src/test-integration/java'
+        }
+        resources {
+            srcDir 'src/test-integration/resources'
+        }
+    }
+}
+test.dependsOn('compileIntegrationTestJava')
+
+configurations {
+    integrationTestImplementation.extendsFrom testImplementation
+    integrationTestRuntimeOnly.extendsFrom testRuntimeOnly
+}
+
 dependencies {
     implementation project(':airbyte-config:models')
-    implementation project(':airbyte-singer')
     implementation project(':airbyte-integrations:base-java')
+    implementation project(':airbyte-singer')
+
+    integrationTestImplementation project(':airbyte-integrations:integration-test-lib')
+    integrationTestImplementation project(':airbyte-integrations:csv-destination')
```

Comment:
why does it need to be a dependency of itself?

Reply:
i'm using a constant from the main module in testImplementation. i can just duplicate it.